src/main/kotlin/net/leanix/vsm/gitlab/broker/webhook/adapter/feign/GitlabWebhookProvider.kt

```diff
@@ -39,21 +39,26 @@ class GitlabWebhookProvider(
         }.getOrThrow()
     }
 
-    override fun createWebhook(): GitlabWebhook {
-        return kotlin.runCatching {
-            webhookClient.createWebhook(
-                url = "$gitlabWebhookUrl$LEANIX_WEBHOOK_PATH",
-                token = leanixId,
-                receivePushEvents = true,
-                receiveTagPushEvents = false,
-                receiveMergeRequestEvents = true,
-                receiveRepositoryUpdateEvents = true,
-                enableSSLVerification = enableSSLVerification
-            )
-        }.onSuccess {
-            logger.info("Webhook created with id ${it.id}")
-        }.onFailure {
-            logger.error("Error creating webhook: ${it.message}")
-        }.getOrThrow()
+    override fun createWebhook(): GitlabWebhook? {
+        return if (gitlabWebhookUrl.isEmpty() || gitlabWebhookUrl.isNotBlank()) {
```

Comment:
Is this correct? If you get an empty webhook url you cannot create a webhook. I think the `isNotBlank()` should be sufficient.

Reply:
fixed in the latest commit.